### DISCUSSION TITLE
"Back to top" localization

### DISCUSSION THREAD

**Discussion by aborruso** [squidfunk/mkdocs-material#3223] [Ideas]
Hi,
I have set my website in Italian, and it seems that there is not localization for "back to top".

What about to add it?

Thank you

![Screenshot_20211114-132147.png](https://user-images.githubusercontent.com/30607/141680880-4be101c2-65e0-4cb1-9ccf-f8b69b6dad35.png)

**Comment by squidfunk** [comment]
~~Sure, good point. Please open an issue!~~ See my comment.

**Reply by squidfunk** [reply]
Wait, no! It is already localized. The Italian translation is just missing. In fact, even more translations are missing. Care to submit a PR with the missing translations?

Italian:

https://github.com/squidfunk/mkdocs-material/blob/9dd23f6926ca5910d4323d87653f4fb3df17369d/src/partials/languages/it.html#L23-L45

English (default):

https://github.com/squidfunk/mkdocs-material/blob/9dd23f6926ca5910d4323d87653f4fb3df17369d/src/partials/languages/en.html#L24-L62

**Reply by aborruso** [reply]
I'll do it.

Thank you very much

**Reply by aborruso** [reply]
> Care to submit a PR with the missing translations?

Today I have added only "back to top": https://github.com/squidfunk/mkdocs-material/pull/3224

I will add all the others.

Thank you

**Reply by squidfunk** [reply]
Okay, thanks. However, I prefer to wait until you added the others, so we can merge it in one go.

**Reply by aborruso** [reply]
> Okay, thanks. However, I prefer to wait until you added the others, so we can merge it in one go.

Done https://github.com/squidfunk/mkdocs-material/pull/3224/commits/1dea6d6ba919c139ffbb7b2064d1fd98ea0b27c8

Thank you

**Comment by jasonren0403** [comment]
I've noticed that most of the updated flags have no corresponding translations catching on 😢

**Reply by squidfunk** [reply]
Yes, I'm afraid I only speak German and English. My Spanish and French are both very rusty 😊 Thus, I'm dependent on users updating the new translations. PRs are always appreciated!
